src/styles/lioshi.css

```diff
@@ -0,0 +1,88 @@
+/* Lioshi Theme */
```

Comment:
This is the line i was talking about.  I don't care how you capitalize your name. :-)  I care about "what is the name of this theme".

Some are purposely lowercase, some aren't. :-)